src/components/MeetingHistory/MeetingHistory.jsx

```diff
@@ -9,39 +10,45 @@ function MeetingHistory() {
 
   return (
     <div>
-      <div 
-        style={{
-          backgroundColor: 'grey',
-          padding: '19px 0px',
-          marginBottom: '7px',
-          boxShadow: '0px 3px 5px rgba(0, 0, 0, .3)'
-        }}
-      >
-        <Typography
-          variant='h4'
+        <Box
```

Comment:
changed the div to a box, added some styling to work with box component and the toolbar